### DISCUSSION TITLE
Pyenv, tk, Fails to launch the GUI, Manjaro Linux

### DISCUSSION THREAD

**Discussion by MrDaisyBates** [bmaltais/kohya_ss#1459] [Q&A]
Hey, guys. I hope you're well and I hope I chose the correct venue for this. 

I'm using Manjaro (Linux) and, to setup Kohya with the correct Python, I needed to use pyenv to change my local python version to an appropriate version. This has seemed to work correctly. 

However, one thing that seems to be problematic is that while I have tk installed on my operating system, due to me using pyenv to shim and route my python to a different location, it seems like tk might just not be installed with the pyenv python. 
![image](https://github.com/bmaltais/kohya_ss/assets/143578183/f538a528-e118-4c62-9b69-8b969bfb5c82)
I did have another issue at the top (that I've not researched yet, but if the answer is simple, I'm happy to hear it). 

What I'm specifically referring to is the stuff towards the bottom. 

I'm also NOT very savvy with coding/programming type stuff, so feel free to talk to me like I'm 5 here ahahaha. 

I think I understand the problem (the pyenv version of python not having tk). But I don't understand what the solution looks like. I'm assuming there's a healthy linux community here, so I'm hoping I'm okay asking this here. 

Thanks in advance!

Edit: I have posted this on the Stable Diffusion Reddit and the pyenv pages. Whichever leads me to a solution, I will edit the solution in here just in case any future internet traveler happens to stumble on it and need it. 
![image](https://github.com/bmaltais/kohya_ss/assets/143578183/a0a53b68-4def-4900-9d52-464029148054)
I did get rid of the LD_LIBRARY_PATH issue by typing the suggested command up there, but the tk issue persists. I say I fixed the path issue, but every time I open a new terminal window it seems the issue comes back, so I must not be doing any permanent solution. 
![image](https://github.com/bmaltais/kohya_ss/assets/143578183/8894652c-3fc1-4dd9-94ff-ecc2e08b0592)
![image](https://github.com/bmaltais/kohya_ss/assets/143578183/a8a0ad91-42e9-4f78-9750-6ffb1bb79e7a)
A user on reddit suggested my answer might lie in this, but I don't understand it:
https://stackoverflow.com/questions/60469202/unable-to-install-tkinter-with-pyenv-pythons-on-macos

**Comment by MrDaisyBates** [comment]
After spending more time trying to understand the problem, my assumption is that the tk installed inside of my pyenv folder (the shim'd version of python)...it's just not configured correctly or something?

Anyways, I don't know if I'll get support here, so I've taken it up on the Manjaro forums hoping that I can maybe get assistance there.
https://forum.manjaro.org/t/tk-not-configured-to-work-with-python-pyenv/147163

**Comment by wutipong** [comment]
Do you have tk installed in the OS? Try running `# pamac info tk` and see if it's installed or not
